recipes/uni-algo/all/conanfile.py

```diff
@@ -0,0 +1,66 @@
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.build import check_min_cppstd
+from conan.tools.files import get, copy
+from conan.tools.layout import basic_layout
+from conan.tools.scm import Version
+import os
+
+required_conan_version = ">=1.52.0"
+
+class UniAlgoConan(ConanFile):
+    name = "uni-algo"
+    description = "Unicode Algorithms Implementation for C/C++"
+    license = ("MIT", "Public Domain")
```

Comment:
```suggestion
    license = ("MIT", "Unlicense")
```
https://github.com/conan-io/conan-center-index/blob/master/docs/adding_packages/conanfile_attributes.md#license-attribute